site/docs/tools/cli.md

```diff
@@ -91,34 +106,34 @@ Usage: nessie cherry-pick [OPTIONS] [HASHES]...
   Transplant HASHES onto current branch.
```

Comment:
"Current" is confusing here. It either merged into the default branch or the explicitly specified branch. Maybe just use "another"?